localstack/http/asgi.py

```diff
@@ -123,6 +124,7 @@ def __init__(
 
         self._more_body = True
         self._buffer = bytearray()
+        self._buffer_file = SpooledTemporaryFile()
```

Comment:
Curious about this change - doesn't seem to be referenced. Do we need it in the context of the extensions?

Reply:
whoops, that's just some leftover experiment